src/Devices/API/planty_routes.py

```diff
@@ -16,3 +16,31 @@
 })
 def add_planty(request, data: PlantyInput):
     return get_planty_model().objects.create_planty(dict(data))
+
+@router.get("/{planty_id}", response={
+    200: PlantyOutput,
+    400: schemas.BadRequestResponse,
+    404: schemas.NotFoundResponse,
+    500: schemas.InternalServerErrorResponse
+})
+def get_planty(request, planty_id: UUID):
+    return get_object_or_404(get_planty_model(), id=planty_id)
```

Comment:
You forgot to put the status code 200 at the time of return.

Reply:
fixing it right now boss